### DISCUSSION TITLE
Can this utilize a MIDI.js soundfont ?

### DISCUSSION THREAD

**Discussion by PJOttCanHome** [jazz-soft/JZZ-gui-Player#3] [Q&A]
I want to use this along with a MIDI.js soundfont.  Is that possible?

I can't see in constructor or options a way to point to the soundfont URL, etc.
(I've enabled JZZ.js, MIDI.js, JZZ.midi.SMF.js, JZZ.synth.MIDIjs.js).

Do I need to chain things together, maybe? Create a synth object first?  I'd need an example (and maybe add that to the example code?)

**Comment by jazz-soft** [comment]
If you register MIDIjs synth before creating the player, the player will find it.
See JZZ.input.MIDIjs.register(...) at https://jazz-soft.net/doc/JZZ/synthMIDIjs.html

**Reply by PJOttCanHome** [reply]
This page should use: JZZ.synth.MIDIjs, not JZZ.input.MIDIjs, right?

https://jazz-soft.net/doc/JZZ/synthMIDIjs.html

**Reply by jazz-soft** [reply]
Thanks a lot for finding the typo! I have uploaded the fixed version.
